dlc_security/musicrooftop.lua

```diff
@@ -0,0 +1,22 @@
+exports('GetMpSecurityMusicRoofTopObject', function()
+    return MpSecurityMusicRoofTop
+end)
+
+MpSecurityMusicRoofTop = {
+    Ipl = {
+        Interior = {
+            ipl = {
+                'sf_billboards', -- -592.6896, 273.1052, 116.302444
```

Comment:
Isn't this and billboards.lua the exact same? Also this isn't being used in fxmanifest

Reply:
Fixed